datalad/support/gitrepo.py

```diff
@@ -448,7 +493,7 @@ def pipe_data_received(self, fd, byts):
                     # it to become a recognizable progress report
                     self._unprocessed = line
 
-    def _parse_progress_line(self, line):
+    def _parse_progress_line(self, line: bytes) -> bool:
```

Comment:
since this is purely internal, lets  make it

```suggestion
    def _parse_progress_line(self, bytes_line: bytes) -> bool:
```

and decode into regular `line` (not `sline`) thus staying more in line with other str line's and minimizing the diff

Reply:
Done: 78e0a3890